packages/bot-web-ui/jest.config.js

```diff
@@ -12,5 +12,16 @@ module.exports = {
         '^Stores/(.*)$': '<rootDir>/src/stores/$1',
         '^Utils/(.*)$': '<rootDir>/src/utils/$1',
     },
+    coveragePathIgnorePatterns: [
+        '<rootDir>/.eslintrc.js',
+        '<rootDir>/jest.config.js',
+        '<rootDir>/coverage/lcov-report',
+        '<rootDir>/dist',
+        '<rootDir>/build',
+        '<rootDir>/docs',
+    ],
```

Comment:
@rupato-deriv 
instead of it we use `collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}', '!**/node_modules/**']`
```suggestion

```

Reply:
removed ✅